wrappers/wrapper.rb

```diff
@@ -693,6 +693,34 @@ def compute_vehicles_capacity(vrp)
       capacities.reject{ |_k, v| v.nil? || v.negative? }
     end
 
+    def reject_services_from_inconsistent_relations(vrp, unfeasible)
```

Comment:
I am still not sure if we need this check or not. Imagine 5 services marked as `same_route`; 3 of them has a common vehicle and 2 of them has another common vehicle -- i.e., we cannot serve all with the same vehicle. 

If we do nothing or-tools will serve either 2 or 3 of these services respecting  `same_route` and unassign/skip the remaining part (3 or 2, respectively). Since or-tools will take into account all the constraints (capacity, duration) and objective function (distance, duration) to select which "group" to plan and which "group" to unassign; we shouldn't do this elimination beforehand.  

With the suggested code, we unassign all services of the same `same_route` relation because of a single incompatible service; but this would make it really hard to use `same_route` and `same_vehicle` relations. It would force the user to make sure all services of a `same_route/vehicle` relation can be "actually" done on the same vehicle. But instead of forcing them we can just do "what we can" and the unplanned part will stay unplanned. (*)

I think we should let or-tools decide who to assign, who to unassign.

* : We might want to do this check as a post-process ! over unassigned services (and check if it was ever possible to process all of the in the same vehicle/route and if that is not the case, add this "reasoning" only to the unassigned ones). This is kind a out-of-scope of this PR (and since we don't have a proper post-processing mechanism yet, it would take too long), we can create a ticket for this -- by copying this code to that ticket (with the suggestions I just made) or giving the link of this comment so that we don't rediscover the wheel.